Libplanet/Tx/RawTransaction.cs

```diff
@@ -13,34 +10,27 @@ namespace Libplanet.Tx
     [Equals]
     internal readonly struct RawTransaction
     {
-        public static readonly byte[] NonceKey =
-            Encoding.ASCII.GetBytes("nonce");
+        public static readonly byte[] NonceKey = { 0x01 };
```

Comment:
Rather than assigning just numbers, how about using a single ASCII character for little meanings?  E.g., `0x6e` (`'n'`) for `NonceKey`.